typescript/ws-protocol/README.md

```diff
@@ -1,43 +1,32 @@
-# Foxglove Studio WebSocket protocol
+# Foxglove WebSocket protocol in TypeScript
 
-This package provides server and client implementations of the [Foxglove Studio WebSocket protocol](https://github.com/foxglove/ws-protocol), enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
-
-The protocol is encoding-agnostic, i.e. it can support Protobuf messages, JSON messages, etc. (as long as the desired encoding is supported by both client and server).
+The Foxglove WebSocket protocol is encoding-agnostic, as long as the desired encoding is supported by both client and server. It currently supports Protobuf and JSON messages.
 
 ## Installation
 
+Install the `ws-protocol` `npm` package:
+
 ```
 $ npm install @foxglove/ws-protocol
 ```
 
-This package does not require a specific WebSocket server or client implementation, so you will need to install your own. For Node.js, you can use the [ws](https://www.npmjs.com/package/ws) package:
+You'll also need to install your own WebSocket server or client implementation. For Node.js, you can use the [ws](https://www.npmjs.com/package/ws) package:
```

Comment:
I don't like this better, I think it's helpful to point out explicitly that this package does not come with a hard dependency on a specific websocket server/client.